Sources/ArcGISToolkit/Components/Offline/MetadataView.swift

```diff
@@ -0,0 +1,128 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct MetadataView: View {
```

Comment:
```suggestion
struct PreplannedMetadataView: View {
```
This view should be specified for the preplanned map model. It may not be re-usable for the on-demand model since that version of the metadata view will support editing the title, description, and map area.